index.js

```diff
@@ -29,8 +29,8 @@ function clipboardCopy (text) {
   // Get a Selection object representing the range of text selected by the user
   var selection = win.getSelection()
 
-  // Fallback for Firefox which fails to get a selection from an <iframe>
-  if (!selection) {
+  // Fallback when iframe's window can not access functions <iframe>
+  if (!win.clipboardCopy) {
```

Comment:
I think it is always `false` and fallbacks to use of the global window.

I believe the `iframe` hack no longer works on Chrome, it is better to be removed from the codebase.

Reply:
This line makes no sense. It will always be false unless you include this library inside the iframe, which no one is doing.